Source/ACE/Entity/Landblock.cs

```diff
@@ -276,7 +276,7 @@ public List<WorldObject> GetWorldObjectsInRange(WorldObject wo, float maxrange,
             return allworldobj;
         }
 
-        public List<WorldObject> GetWorldObjectsByGuid(ObjectGuid objectguid, bool neighbors)
+        private List<WorldObject> GetWorldObjectsByGuid(ObjectGuid objectguid, bool neighbors)
         {
             List<WorldObject> allworldobj = new List<WorldObject>();
             lock (objectCacheLocker)
```

Comment:
Copying the list, then looking for 1 object is a hell of a lot slower than just looking for the 1 while you have the lock.